tensorflow/c/c_api.h

```diff
@@ -964,7 +964,7 @@ typedef struct TF_Library TF_Library;
 // Pass "library_filename" to a platform-specific mechanism for dynamically
 // loading a library. The rules for determining the exact location of the
 // library are platform-specific and are not documented here.
-// Expects the symbols "RegisterOps", "RegisterKernels", and "GetOpList", to be
+// Expects the symbols "RegisterKernels", and "GetOpList", to be
```

Comment:
I think you probably want to leave out this entire line... look at tensorflow/core/framework/load_library.cc::LoadLibrary comment